admin/fleetcommander/fcdbus.py

```diff
@@ -198,43 +199,50 @@ def run(self, sessionbus=False):
         self._loop = GObject.MainLoop()
 
         # Prepare changes listener
-        self.changeslistener_port = self.get_free_port()
-        self.changeslistener = Soup.Server()
-        self.changeslistener.listen_all(
-            self.changeslistener_port, Soup.ServerListenOptions.IPV4_ONLY)
-        self.changeslistener.add_handler(
+        self.webservice_port = self.DEFAULT_WEBSERVICE_PORT
+        self.webservice = Soup.Server()
+        try:
+            self.webservice.listen_all(
+                self.webservice_port, Soup.ServerListenOptions.IPV4_ONLY)
+        except Exception, e:
+            logging.error('Error starting webservice: %s' % e)
+            sys.exit(1)
+        self.webservice.add_handler(
             '/changes/submit/', self.changes_listener_callback)
 
+        self.webservice.add_handler(
+            '/clientdata/', self.client_data_callback)
```

Comment:
Nope. We don't need it, but for retaining old path and for having a common pattern for data serving, I preferred to use it.